docs/central-install-digital-ocean.rst

```diff
@@ -565,6 +565,104 @@ DKIM is a security trust protocol which is used to help verify mail server ident
 
      $ docker compose build mail && docker compose up -d mail
 
+.. _central-install-digital-ocean-sso:
+
+Enabling Single Sign-on
+-----------------------
+
+By default, users log into Central using an email address and password. However, if Single Sign-on (SSO) is enabled, then Central will no longer manage users' passwords and will instead forward users to a separate login server. If all the users of your Central installation already have an account on another site, it may be possible to configure Central so that users can use that account to log into Central. Examples of this include Google Workspace and Azure Active Directory. Under this setup, the login server is called the "identity provider." If SSO is enabled, the identity provider will manage users' passwords, not Central.
+
+The identity provider may enforce different requirements for login than Central does on its own. For example, Central requires that new passwords are at least 10 characters, but it does not require other password characteristics, such as the presence of certain symbols. However, if SSO is enabled in Central, and if the identity provider is configured to require specific password characteristics, then users will need to fulfill those requirements in order to log into Central. As another example, on its own, Central does not support two-factor authentication. However, if SSO is enabled, and if the identity provider is configured to require two-factor authentication, then users will need to complete two-factor authentication before logging into Central.
+
+Central is not compatible with every identity provider. Central uses the OpenID Connect (OIDC) protocol. When SSO is enabled in Central, Central does not manage passwords, but it still identifies users using their email address. Central assumes that the identity provider verifies email addresses, requiring users to prove ownership of the email address they specify. If that is not the case, then do not enable SSO in Central.
+
+If SSO is enabled, some features in Central will stop working. OData feeds will not be accessible in tools like Power BI or Excel. The API will not be available, which means that pyODK and ruODK will not be able to fetch data from Central. In the future, we hope to provide a way to access OData and the API when SSO is enabled.
+
+To enable SSO in Central, you will need to provide information from your identity provider, specifically the issuer URL, client ID, and client secret. If you use Google Workspace or Azure Active Directory, you can follow the instructions below to find this information.
```

Comment:
Let's add these sections as part of a separate PR after release as you suggest below.